parachains/runtimes/collectives/collectives-polkadot/src/constants.rs

```diff
@@ -13,6 +13,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+pub const POLKADOT_TREASURY_PALLET_INDEX: u8 = 19;
```

Comment:
LGTM: https://github.com/paritytech/polkadot/blob/5a09bf62ec7be0e90dfae335a80459728a896d95/runtime/polkadot/src/lib.rs#L1429

Reply:
This index can change in a runtime upgrade (although we hope it won't). The `AccountId` is generated from the [module ID](https://github.com/paritytech/polkadot/blob/5a09bf62ec7be0e90dfae335a80459728a896d95/runtime/polkadot/src/lib.rs#L809), not the pallet index. So I wonder if it's better to derive the same `AccountId` from that?